common/lib/capa/capa/templates/jstextline.html

```diff
@@ -19,7 +19,7 @@
   % endif
 
   <span class="status ${status.classname}" id="status_${id}" aria-describedby="input_${id}">
-    <span class="sr">${status.display_name</span>
+    <span class="sr">${status.display_name}</span>
```

Comment:
Once the file was being processed by Mako, it was a very obvious syntax error stack trace :)